src/Sniffer.php

```diff
@@ -39,9 +39,15 @@ public function preCommit()
 
         echo 'Running PHP CodeSniffer in ' . $this->root . PHP_EOL;
         $sniffer = new PHP_CodeSniffer_CLI();
+        @$config = $this->getExtraKey('php-composter-phpcs-wpcs', [
```

Comment:
Please avoid the "shut-up" operator (`@ `). The `getExtraKey()` already verifies that the key does indeed exist and falls back to the value you provide if not. Any other error should either be dealt with separately, or bubble up because it might be pointing to a bigger issue.
```suggestion
        $config = $this->getExtraKey('php-composter-phpcs-wpcs', [
```